common/src/api/external/mod.rs

```diff
@@ -2718,6 +2733,43 @@ pub struct BgpPeerStatus {
     pub switch: SwitchLocation,
 }
 
+/// Opaque object representing BGP message history for a given BGP peer. The
+/// contents of this object are not yet stable.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct BgpMessageHistory(mg_admin_client::MessageHistory);
```

Comment:
I made this newtype to hide the details of the Maghemite data structure with the opaque `JsonSchema` implementation below.

Reply:
I think this is a big improvement, and we have the option of providing more details later. This data is roughly like log message or debug output so it seems reasonable that we might choose not to specify it in great detail.

If you're happy with this so am I.